docs/postprocessors.rst

```diff
@@ -27,6 +27,11 @@ Perun's tool suite currently contains the following two postprocessors:
      which can be used for further postprocessing (e.g. by regression analysis) or to group
      similar amounts of resources.
 
+  4. :ref:`postprocessors-regressogram` (authored by **Simon Stupinsky**) same as above
+     mentioned *regression analysis*, attempts to find the fitting model for the dependent variable
+     based on another independent one. Regressogram represents the simplest method of non-parametric
```

Comment:
Seems wrong: you are not finding the fitting model based on another independent one. Quite contrary we are trying to fit models through data only. Definitely think if this first sentence is true :-)

Btw. I think there should be comma before same

Reply:
Yes, they were nonsense. I wrote it again, and I think it should be cool. Look at it, please.